plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -61,10 +61,10 @@ func calculateResourceOccupancy(pod *api.Pod, node api.Node, nodeInfo *scheduler
 	cpuScore := calculateScore(totalMilliCPU, capacityMilliCPU, node.Name)
 	memoryScore := calculateScore(totalMemory, capacityMemory, node.Name)
 	glog.V(10).Infof(
-		"%v -> %v: Least Requested Priority, Absolute/Requested: (%d, %d) / (%d, %d) Score: (%d, %d)",
+		"%v -> %v: Least Requested Priority, Absolute/Requested(CPU:millicores, memory:bytes): (%d, %d) / (%d, %d) Score: (%d, %d)",
```

Comment:
OK. That's reasonable, I will modify as you wrote above. BTW, could you please also review #26520 , #26774? Thanks @davidopp :smile: